### DISCUSSION TITLE
我的umi项目开启了MFSU后，浏览器中开启控制台热更新慢

### DISCUSSION THREAD

**Discussion by cqg001** [umijs/umi#12433] [Q&A]
我的umi项目开启了MFSU,umi版本是3.5.41，我发现我的项目在开控制台的时候热更新特别慢，但是我关闭控制台之后它就非常快，我目前想都快，无论我是否f12开了控制台

**Comment by fz6m** [comment]
umi 3 不维护了，建议升级到 umi 4 。

umi 3 中的 MFSU 是没有足够完善的，如需使用更好的 MFSU ，请升级到 umi 4 。
